src/app/static-pages/trust-checking/trust-checking-page/trust-checking-page.component.scss

```diff
@@ -0,0 +1,172 @@
+  font-size: 36px;
+  line-height: initial;
+
+  @media screen and(min-width: $breakpoint-l) {
+    font-size: 52px;
+  }
+}
+.section-background {
+  width: 100vw;
+  background-color: colors.$color__white;
+  justify-content: center;
+
+  display: flex;
+  flex: 1 1 auto;
+
+  padding: 0 10px;
+  @media screen and(min-width: 800px) {
+    padding: 0;
+  }
+
+  app-breadcrumb {
+    align-self: flex-start;
+  }
+  &-grey-edges {
+    @extend .section-background;
+    background-image: url(/assets/images/backgrounds/grey-edgy-background.svg);
+    background-size: cover;
+    background-repeat: no-repeat;
+    padding: 100px 10px;
+    @media screen and(min-width: 800px) {
+      padding: 100px 0;
+    }
+  
+    margin-top: -20px;
+  }
+}
+.intro {
+  display: flex;
+  flex-direction: column;
+  &-info {
+    display: flex;
+    flex-direction: column;
+
+    @media screen and(min-width: 800px) {
+      flex-direction: row;
+      justify-content: space-between;
+      gap: 16px;
+    }
+  }
+  .image {
+    margin-top: 16px;
+    align-self: center;
+
+    &-mobile {
+      width: 80%;
+      @media screen and(min-width: 800px) {
+        display: none;
+      }
+    }
+    &-desktop {
+      display: none;
+      @media screen and(min-width: 800px) {
+        display: flex;
+        width: 50%;
+        max-height: 500px;
+        order: 1;
+      }
+    }
+  }
+}
+.text {
+  @include fonts.font__effra--regular;
+  color: colors.$color__mysterious-blue;
+  font-size: 18px;
+
+  &-bold {
+    @include fonts.font__effra--bold;
+    color: colors.$color__mysterious-blue;
+  }
+  &-narrow {
+    max-width: 580px;
+  }
+}
+
+.section-head {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  padding: 25px 0;
+
+  .section-heading {
+    @include fonts.font__pt-mono--bold;
+    font-size: 21px;
+    text-align: center;
+    // line-height: 0px;
+  
+    &-supernova {
+      color: colors.$color__supernova;
+    }
+  }
+  .section-sub-heading {
+    @include fonts.font__effra--bold;
+    font-size: 30px;
+    text-align: center;
+    // line-height: 0px;  
+  
+    @media screen and (min-width: $breakpoint-m) {
+      font-size: 36px;
+    }
+  }
+}
+
+.about-snippet {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  max-width: 820px;
+  margin: auto;
+  margin-top: 20px;
+
+  @media screen and(min-width: 800px) {
+    flex-direction: row;
+    gap: 40px;
+  }
+
+  &-alt {
+    @extend .about-snippet;
+
+    @media screen and(min-width: 800px) {
+      flex-direction: row-reverse;
+    }
+  }
+
+  .illustration {
+
+    img{
+      max-height: 150px;
+      max-width: 150px;
+  
+    }
+  }
+
+  .title {
+    @include fonts.font__effra--bold;
+    font-size: 36px;
+    line-height: normal;
+    margin: 0;
+  }
+
+  .text {
+    @include fonts.font__effra--regular;
+    font-size: 18px;
+  }
+}
+.criteria-grid {
+  @media screen and(min-width: 800px) {
```

Comment:
Zwischen 800px und 1024 fehlt rechts und links ein wenig Abstand zum Rand.
Evtl. könntest du aber auch einfach in dem Bereich zwei statt drei Spalten nutzen.

![image](https://user-images.githubusercontent.com/38329766/148692525-4d3be742-a742-459a-84ab-cf4d66ccaa56.png)

Reply:
Betrifft auch den Textteil, fällt mir gerade auf.

Reply:
Der Vorschlag mit zwei Spalten würde auch das hier lösen:

![image](https://user-images.githubusercontent.com/38329766/148692555-eb7431d0-4c77-4f8b-800e-f6c07d5eff93.png)

Reply:
Gute Idee, erledigt.

Reply:
Zwischen 800px und 820px hast jetzt noch den Fall, dass der Textteil keinen Abstand zum Rand hat,

![image](https://user-images.githubusercontent.com/38329766/148698379-816fcad6-15ed-49e6-92c0-2a0e29421aa2.png)

Reply:
Und zwischen 1024px und 1350px ist aktuell noch der Header im letzten Criterion etwas gedrungen.

![image](https://user-images.githubusercontent.com/38329766/148698514-cffb2d49-3d17-4469-a77b-c628db8503fb.png)